main.go

```diff
@@ -88,6 +89,7 @@ var (
 	clientAllowedIPs     = clientCommand.Flag("verify-ip", "").Hidden().PlaceHolder("SAN").IPList()
 	clientAllowedURIs    = clientCommand.Flag("verify-uri", "Allow servers with given URI subject alternative name (can be repeated).").PlaceHolder("URI").Strings()
 	clientDisableAuth    = clientCommand.Flag("disable-authentication", "Disable client authentication, no certificate will be provided to the server.").Default("false").Bool()
+	clientQuiet          = clientCommand.Flag("client-quiet", "Silence client logging about TLS errors").Default("false").Bool()
```

Comment:
We can just call this flag "--quiet" for both server and client, is more consistent that way and matches what we do for other flags.

Reply:
OK, changed.